Laravelインストール

kthiro opened the pull request:
vagrant上で Laravel 環境を構築

kthiro committed:
Laravelインストール
vagrant上に
* nginx
* PHP
* MySQL
をインストールした後、
* composer
をインストールし、composerを使用して
* Laravel
をインストール。vagrant上のUbuntu-16.04上（ゲストOS上）でのインストール手順を
* install_guestos.sh
へ記述。

kthiro committed:
Laravelインストール
* 開発環境の概要について記述

orihara-ori commented on README.md:
```diff
@@ -0,0 +1,20 @@
+* MySQL バージョン 5.7.25
+* PHP バージョン 7.0.33
+* composer バージョン 1.8.4
+* Laravel バージョン 2.0.1
```
さすがに、今、インストールすると、バージョン2.0.1ってことはないんじゃないかな？と
https://readouble.com/laravel/

kthiro committed:
Laravelインストール
* install_guestos.sh
LaravelのインストールにはPHP7.1.3以上が必要だったため、ゲストOSでの環境構築手順を修正。vagrant上に構築したubuntu-16.04のデフォルトのパッケージのままPHPをインストールするとバージョンが7.0.Xであるため、外部リポジトリであるppa:ondrej/phpを使用してPHP7.3.Xをインストール。

* README.md
ゲストOSの環境構築手順を修正認め、READEME.mdも修正

orihara-ori commented:
:ok:
